client/web/src/site/backend.ts

```diff
@@ -47,6 +45,11 @@ export function refreshSiteFlags(): Observable<never> {
                         noLicenseWarningUserCount
                     }
                 }
+                fragment AlertFields on Alert {
```

Comment:
Let's rename it to be more descriptive 
```suggestion
                fragment SiteFlagAlertFields on Alert {
```